quinn-proto/src/tests/mod.rs

```diff
@@ -2750,19 +2750,3 @@ fn reject_new_connections() {
     pair.server.assert_no_accept();
     assert!(pair.client.connections.get(&client_ch).unwrap().is_closed());
 }
-
-#[test]
-fn reject_remote_address() {
```

Comment:
Can we easily test that explicitly rejecting a connection produces the expected error (i.e. not a timeout) on the client side? Coverage at the `quinn` layer is good, and adequate if a unit test is difficult for some reason, but unit tests have many fewer variables to worry about.

Reply:
Test re-added, renamed to `reject_manually`, and now asserts that the correct error occurs.